server/checkdoc.js

```diff
@@ -0,0 +1,28 @@
+const Doc = require('doc');
+const Taboo = require('taboo');
+
+const inputdoc = (req, res) => {
+  let docinst = new Doc({
+    title: req.body.name,
+    content: req.body.text,
+    owner_id: req.body.owner_id,
+    locked: false,
+    revision_id: 0,
+  });
+
+  let taboo_words_data = Taboo.find({}, function (err, results) {
```

Comment:
model.find does not return the results. Please read documentation on how to properly use this method

Reply:
changed to send results from response